packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -171,6 +171,28 @@ describe('multi-domain', { experimentalSessionSupport: true, experimentalMultiDo
       cy.switchToDomain()
     })
 
+    it('fails on a bad domain name', (done) => {
+      cy.on('fail', (err) => {
+        expect(err.message).to.equal('`cy.switchToDomain()` requires the first argument to be a string representing a domain that does not not include any sub domains. You passed: `foobar`')
+
+        done()
+      })
+
+      cy.switchToDomain('foobar', () => {
+      })
```

Comment:
more of a nit here, but figure we don't need the function body?
```suggestion
      cy.switchToDomain('foobar', () => undefined)
```